src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs

```diff
@@ -280,13 +280,17 @@ public async Task CancelDisputeAsync(long disputeId, string cancelledReason, Cla
             GetUserName(user));
         await _bus.PublishWithLog(_logger, fileHistoryRecord, cancellationToken);
 
-        // Publish file history of cancellation email
-        fileHistoryRecord.AuditLogEntryType = FileHistoryAuditLogEntryType.EMCA;

```

Comment:
I think this file history record is specific for logging automated cancellation email sent to citizen so it should not be removed.

Reply:
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/140637695/8ec487ff-408b-4fea-b207-405915d9cc02)
Already we have the event, it's just getting duplicated

Reply:
That `EMCA` Audit Log Entry Type is not published anywhere else in the application though. I don't think that it can be duplicated unless it's an issue with displaying it. That message is different than `Staff have cancelled the dispute, status of dispute changed to Cancelled` log which is `SCAN` type.

Reply:
![image](https://github.com/bcgov/jag-traffic-courts-online/assets/140637695/87a0ab8f-ac61-457b-b30d-8e70fec4e2f2)
It's not duplicated but the message is same, clients also do not want to show that message. If you see in the screenshot, the 2nd and 3rd row means the same thing

Reply:
Yeah the third message mentions "Email Sent" in its row so that also indicates the cancelled email is sent I guess. Ok then, if client does not want to see that message, it's okay to remove.